scripts/lib/image_defs

```diff
@@ -0,0 +1,7 @@
+REPO="quay.io/submariner"
+
+declare -A project_images
+project_images[lighthouse]="lighthouse-agent lighthouse-coredns"
+project_images[shipyard]="nettest shipyard-dapper-base"
+project_images[submariner]="submariner submariner-globalnet submariner-route-agent"
```

Comment:
```suggestion
project_images[submariner]="submariner submariner-globalnet submariner-route-agent submariner-networkplugin-syncer"
```
new kid on the block ':D

Reply:
Oh wait, that one we don't publish it yet from the submariner repo, I guess we need to update the push action

Reply:
Lets update it once its being published